recipes/cppcheck/all/conanfile.py

```diff
@@ -63,7 +64,6 @@ def package(self):
         self.copy("*", dst=os.path.join("bin","cfg"), src=os.path.join(self._source_subfolder,"cfg"))
```

Comment:
```suggestion
        self.copy("*", dst=os.path.join("bin"), src=os.path.join(self._source_subfolder,"cfg")) # cppcheck looks in current directory first
```

Reply:
I tried locally and unfortunately this didn't resolve the problem. However, I digged a bit into the available CMake options and found that `FILESDIR` directs where to install the cfg files. I know updated the PR and install the files to `bin/Cppcheck`